packages/operators/src/common/uuid.js

```diff
@@ -14,10 +14,33 @@
   limitations under the License.
 */
 
-import { v4 as uuidv4 } from 'uuid';
+import { v1, v2, v3, v4, v5 } from 'uuid';
+import { type } from '@lowdefy/helpers';
 
-function _uuid() {
-  return uuidv4();
+function _uuid({ params, location }) {
+  if (type.isNone(params) || params === true) {
+    return v4();
+  }
+  if (params === 'v1') {
+    return v1();
+  }
+  if (params === 'v2') {
+    return v2();
+  }
+  if (params === 'v3') {
+    return v3();
+  }
+  if (params === 'v4') {
+    return v4();
+  }
+  if (params === 'v5') {
+    return v5();
```

Comment:
I think this will fail, as V3/V5 UUIDs require two parameters to work -- they're basically salted hashes (see [the docs of uuidjs/uuid](https://github.com/uuidjs/uuid#uuidv5name-namespace-buffer-offset)).

Reply:
Thanks for catching this. I'll have a look!